net-misc/dibbler/files/dibbler-1.0.1-clang-16-musl-fix.patch

```diff
@@ -0,0 +1,63 @@
+From: Brahmajit Das <brahmajit.xyz@gmail.com>
+Date: Wed, 31 May 2023 06:27:08 +0000
+Subject: [PATCH] Fix build with clang 16
+
+Especially on musl libc
```

Comment:
This is missing a Bug: tag and an explanation of what the actual problem and fix is.

Reply:
Thanks! Fixing